app/src/components/account/UserMenuAddress.vue

```diff
@@ -30,12 +30,14 @@
 </template>
 <script>
 import Address from "common/Address"
+import TmFormMsg from "src/components/common/TmFormMsg"
```

Comment:
what does this look like? can you share a screenshot?

Reply:
:sweat_smile: just by looking at the code you the component is not registered, don't you?

I'm surprised this is taking so long to be merged, such a simple bug :shrug:

Reply:
![image](https://user-images.githubusercontent.com/40721795/91049412-73d4fe00-e61d-11ea-986c-8510c05df948.png)

Reply:
i get it - i'm just surprised we're using this component for this - seems totally unnecessary.